topic_name: '/acoustic_magic_doa/data_raw'
serial_port: '/dev/ttyUSB0'
